site/docs/spark-structured-streaming.md

```diff
@@ -26,6 +26,25 @@ As of Spark 3.0, DataFrame reads and writes are supported.
 |--------------------------------------------------|----------|------------|------------------------------------------------|
 | [DataFrame write](#writing-with-streaming-query) | ✔        | ✔          |                                                |
 
+## Streaming Reads
+
+Iceberg supports processing incremental data in spark structured streaming  jobs which starts from a historical timestamp:
```

Comment:
Typo: double space.

Reply:
I don't quite understand.  Delete the blank line?  This blank line seems to be needed.

Reply:
There are two spaces between "streaming" and "jobs"

Reply:
Sorry for my carelessness.

Reply:
Not a problem! This is why we review.